### ISSUE TITLE
[FEATURE REQUEST]: Review / Remove obsolete code

### ISSUE THREAD

**Issue by smk762** [KomodoPlatform/komodo-wallet-desktop#1803]
As this project has matured we have used a few different price services (e.g. band oracle, direct apis, our own price service).
I"m not sure how many, if any of these are still in effect unnecessarily, but if there are some we can remove it should make things more efficient.
Unless there is a compelling reason to keep it, we should also purge old code related to this e.g. https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/src/core/atomicdex/services/price/oracle/band.provider.cpp which might reduce size of build binaries.

**Comment by smk762** [comment]
related logs:

```
[18:03:07] [info] [band.provider.cpp:54] [767731]: start fetching oracle
[18:03:07] [error] [cpprestsdk.utilities.cpp:51] [767759]: pplx task error: Error resolving address
```

**Comment by cipig** [comment]
are these logs also related?
```
[22:03:21] [warning] [main.prerequisites.hpp:92] [2228575]: Symbol not found for KMD/USD
[22:03:21] [warning] [main.prerequisites.hpp:92] [2228575]: Symbol not found for USD/KMD
[22:03:21] [warning] [main.prerequisites.hpp:92] [2228575]: Symbol not found for KMD/USD
[22:03:21] [warning] [main.prerequisites.hpp:92] [2228575]: Symbol not found for USD/KMD
[22:03:21] [debug] [main.prerequisites.hpp:86] [2228575]: Wallet: Loading chart for BINANCE:KMDUSD
```
they show up when switching to wallet

**Comment by smk762** [comment]
Possibly related: https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/847 (old language selection code)

**Comment by smk762** [comment]
Current PRs that will be merged for 0.5.8 will remove:
- old segwit switch code https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/2280
- old band oracle code https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/2277

**Comment by smk762** [comment]
closing as complete
